airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/TestDestination.java

```diff
@@ -299,6 +302,20 @@ private void runSync(JsonNode config, List<AirbyteMessage> messages, AirbyteCata
     messages.forEach(message -> Exceptions.toRuntime(() -> target.accept(message)));
     target.notifyEndOfStream();
     target.close();
+
+    if (!implementsBasicNormalization())
```

Comment:
```suggestion
        if (!config.get("basic_normalization").asBoolean())
```
This is how the workers decide which normalization step to run, so I'd prefer we use it over the method you currently have to run this. It's closer to reality because it means we will only run normalization when it is actually configured to run (not every time even if it is not configured.)

Reply:
fixed in the PR i merged into this branch.